src/status_im/subs/wallet/wallet_test.cljs

```diff
@@ -367,52 +376,55 @@
            (assoc-in [:wallet :current-viewing-account-address] "0x2")
            (assoc-in [:wallet :networks] network-data)))
     (is
-     (= (list
-         {:path                      "m/44'/60'/0'/0/0"
-          :emoji                     "😃"
-          :key-uid                   "0x2f5ea39"
-          :address                   "0x1"
-          :wallet                    false
-          :name                      "Account One"
-          :type                      :generated
-          :watch-only?               false
-          :chat                      false
-          :test-preferred-chain-ids  #{5 420 421613}
-          :color                     :blue
-          :hidden                    false
-          :prod-preferred-chain-ids  #{1 10 42161}
-          :network-preferences-names #{:mainnet :arbitrum :optimism}
-          :position                  0
-          :clock                     1698945829328
-          :created-at                1698928839000
-          :operable                  :fully
-          :mixedcase-address         "0x7bcDfc75c431"
-          :public-key                "0x04371e2d9d66b82f056bc128064"
-          :removed                   false
-          :tokens                    tokens-0x1}
-         {:path                      ""
-          :emoji                     "🎉"
-          :key-uid                   "0x2f5ea39"
-          :address                   "0x3"
-          :wallet                    false
-          :name                      "Watched Account 1"
-          :type                      :watch
-          :watch-only?               true
-          :chat                      false
-          :test-preferred-chain-ids  #{0}
-          :color                     :magenta
-          :hidden                    false
-          :prod-preferred-chain-ids  #{0}
-          :network-preferences-names #{}
-          :position                  2
-          :clock                     1698945829328
-          :created-at                1698928839000
-          :operable                  :fully
-          :mixedcase-address         "0x7bcDfc75c431"
-          :public-key                "0x"
-          :removed                   false
-          :tokens                    tokens-0x3})
-        (rf/sub [sub-name])))))
+     (=
```

Comment:
Maybe back then we didn't have the `match?` clause available. Since you touched this part, could you change from `=` to `match?` please.

Reply:
For sure 👍 
Updated here: https://github.com/status-im/status-mobile/pull/20636/commits/010ced0d1f085b49595b5db7826fb0efa7592bc3